File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -374,6 +374,8 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-and-Replace-in-Powerpoint-using-Regex).
 
+N> When searching for items within a Master slide in the Presentation document, it is recommended to use the IPresentation.FindAll API instead of the ISlide.FindAll API. 
```

Comment:
When searching for text within a master slide in the presentation, it is recommended to use the IPresentation.FindAll API instead of the ISlide.FindAll API.


>>>>>>>>>>>>>

Add hyperlinks for APIs

Reply:
Note content is modified as per your susgestion and add Add hyperlinks for APIs